package.json

```diff
@@ -110,10 +110,10 @@
   "optionalDependencies": {
     "@rollup/rollup-linux-x64-gnu": "4.14.0"
   },
-  "packageManager": "npm@10.7.0",
+  "packageManager": "npm@10.9.0",
   "engines": {
     "node": ">=18.17.0",
-    "npm": ">=8.5.0"
+    "npm": ">=10.9.0"
```

Comment:
@jacekradko afaik Node 18.17.0 doesn't ship with npm 10.9.0, so in theory if you install that you can't run it or have to update your npm version